spec/index.bs

```diff
@@ -282,10 +282,12 @@ interface GPUShaderStageBit {
 enum GPUBindingType {
     "uniform-buffer",
     "dynamic-uniform-buffer",
-    "sampler",
-    "sampled-texture",
     "storage-buffer",
-    "dynamic-storage-buffer"
+    "dynamic-storage-buffer",
+    "readonly-storage-buffer",
+    "dynamic-readonly-storage-buffer",
```

Comment:
I think it would make sense to move the "dynamic" flag as another `boolean` field near the `GPUBindingType` instead of combinatorically exploding the binding types (think about what happens if/when we add texel buffer views...)

Reply:
Done.